website/source/layouts/docs.erb

```diff
@@ -268,8 +268,8 @@
             <a href="/docs/auth/approle.html">AppRole</a>
           </li>
 
-          <li<%= sidebar_current("docs-auth-aws-ec2") %>>
-            <a href="/docs/auth/aws-ec2.html">AWS EC2</a>
+          <li<%= sidebar_current("docs-auth-aws") %>>
+            <a href="/docs/auth/aws-ec2.html">AWS</a>
```

Comment:
Is there a reason not to move/update this page? I can ask around here if there is an SEO reason but usually we'd handle that at a higher level with redirects.